content/articles/creating-a-simple-chat-app-in-react-using-the-chat-engine-api/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Creating a Simple Chat App in React.js using the Chat Engine API
+---
+
+
+Ever thought of coming up with a chat application, but going the Node.js and or the socket.io way seemed pretty cumbersome for someone solely equipped with front-end skills like React.js? 
+Well, this article fits you right. 
+The reader is brought to an understanding of the Chat engine API that we will be using, to come up with a precise yet functional Direct-Messaging application built in React.js. 
+Approximating to other APIs that deliver to the system, responses from users, and sends back responses of the system to the user, the Chat engine API does this also only with better functionalities.
+
+<!--more-->
+This tool saves you a lot of implementation time you would otherwise need on many lines of code.
+Chat Engine API can implement; 
+ - A group chat application.
+ - Direct messages chat application that would work like Facebook messenger .
+
+### Prerequisites
+You will need to be equipped with what follows next before jumping right to it.
+1. Primary React.js library mastery. Head to [the official documentation](https://reactjs.org/tutorial/tutorial.html) if there's a need to familiarize yourself with React.js.
```

Comment:
This is quite wordy, simplify your sentences where applicable throughout your article.